Tests/GRPCHealthServiceTests/HealthTests.swift

```diff
@@ -285,6 +285,75 @@ final class HealthTests: XCTestCase {
       }
     }
   }
+
+  func testListServices() async throws {
+    try await withHealthClient { (healthClient, healthProvider) in
+      let message = Grpc_Health_V1_HealthListRequest()
+
+      // Empty case
+      try await healthClient.list(request: ClientRequest(message: message)) { response in
+        let statuses = try response.message.statuses
+        XCTAssertEqual(statuses, [:])
+      }
+
+      // Service descriptors and their randomly generated status.
+      let testServiceDescriptors: [(ServiceDescriptor, ServingStatus)] = Array(0 ..< 10).map { i in
+        (
+          ServiceDescriptor(package: "test", service: "Service\(i)"),
+          Int.random(in: 0 ... 1) == 0 ? .notServing : .serving
```

Comment:
This has a better spelling:

```suggestion
          Bool.random() ? .notServing : .serving
```